vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -37331,7 +37331,7 @@ unit:PERCENT-PER-HR
   qudt:conversionMultiplierSN 2.777777777777777777777777777777778E-6 ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T-1D0 ;
   qudt:hasQuantityKind quantitykind:Frequency ;
-  qudt:symbol "%/day" ;
+  qudt:symbol "%/hr" ;
```

Comment:
Actually, the symbol for "hour" was just adjusted in PR #899 to be "h".

Reply:
Thanks - updated the pull request so its /h